data/remnant/remnant missions.txt

```diff
@@ -4531,6 +4531,9 @@ mission "Remnant: Expanded Horizons Storms 1"
 			label readyforstorms
 			`	"I will settle into a bunk to do some pre-calculations. Please let me know when we arrive in Aescolanus."`
 				accept
+	on waypoint
+		require "Research Laboratory"
```

Comment:
Didn't we talk about this already?

Reply:
We might have, but a lot of our conversations blur together after a few weeks/months. 

I'm guessing I need to take this out?

Reply:
The issue with this is that the player doesn't actually need the outfit requirement to finish the waypoint objective. All this does is prevent the dialog from showing up.

Reply:
Yeah, I figured. 

And we don't have a `to waypoint` yet (at least according to the wiki) that I could put such a restriction in.

Anyway, it has been removed, as I'm sure you saw.